docs/help/support-guidelines.md

```diff
@@ -1,5 +1,86 @@
 # Support guidelines
 
+This page explains the support mechanisms we provide.
+
 !!! warning
 
-    Under Construction
+    Before asking any help, search and read this documentation site carefully.
+    Also, understand and follow our [discussion guidelines](../contributing/discussion-guidelines/index.md) during your discussions.
```

Comment:
```suggestion
    Also, please follow the [discussion guidelines](../contributing/discussion-guidelines/index.md) for the discussions.
```

Reply:
@kenji-miyake https://developers.google.com/style/person?hl=en It's described in the last part paragraph here.

Reply:
Great, thank you!
I might have to update other parts.

Reply:
Will fix them later in #51.

Reply:
You can apply the changes here too.

Reply:
I'm not sure if I need `the` for `for the discussions.`
I'd like to wait for a comment from @LalithVipulananthan here, too.

Reply:
@kenji-miyake i removed the last the.